src/MainWindow.xaml

```diff
@@ -2,23 +2,30 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         Title="Cupboard" WindowState="Maximized" WindowStartupLocation="CenterScreen"
-        Background="{StaticResource PrimaryBrush}">
+        Background="{StaticResource PrimaryBrushA}">
 
     <Grid>
-        <!-- Background Image -->
-        <Image Source="pack://application:,,,/Cupboard_Logo.png" Stretch="UniformToFill" Opacity="0.1"/>
-
         <!-- Banner with increased height and bottom outline -->
-        <Border Background="{StaticResource SecondaryBrush}" Height="120" VerticalAlignment="Top" BorderBrush="Transparent" BorderThickness="0">
-            <TextBlock Text="Welcome to Cupboard" Foreground="{StaticResource TertiaryBrush}" HorizontalAlignment="Center" VerticalAlignment="Center" FontSize="32" FontWeight="Bold"/>
+        <Border Background="{StaticResource PrimaryBrushB}" Height="120" VerticalAlignment="Top" BorderBrush="Transparent" BorderThickness="0">
+            <TextBlock Text="Welcome to Cupboard" Foreground="{StaticResource SecondaryBrushB}" HorizontalAlignment="Center" VerticalAlignment="Center" FontSize="32" FontWeight="Bold"/>
         </Border>
 
-        <!-- Login Button in the bottom third of the screen -->
-        <StackPanel VerticalAlignment="Bottom" Margin="0,0,0,150">
+        <!-- Image and Login Button Section -->
+        <StackPanel VerticalAlignment="Bottom" Margin="0,200,0,100" HorizontalAlignment="Center">
+            <!-- Image Above Login Button -->
```

Comment:
So it seems like the image is cutoff at the top. Could you fix this by adding margin to the top?
![image](https://github.com/user-attachments/assets/23941501-844a-45bc-b13f-0b1824999719)

Reply:
I'll move the button down and make the pic smaller.

Reply:
Also, for the previous questions/comments:
1. Warnings: These are compiler warnings that have been there for a while. Don't worry, the warnings displayed are mitigated in other parts of the code. Theses warnings have been there since the first sprint. They are of no concern. 
2. Mock Error: Do you get this on login? If you wanna run mock, make sure the env files are not in Auth0 and Backend folders.

Reply:
I think you can keep the button where it is, just down the size of the image